### PR TITLE
Updating Date & Time Picker button states to match Button

### PR DESCRIPTION
**Pull request by kimsealls**
## Description

Updated states to match button.

## Motivation and Context

The style for these buttons should match the rest of the new design updates.

## How Has This Been Tested?

Tested in Controls Test App

Fixes: #5066

### CODE REVIEW

**Comment by mdtauk** [comment]
I guess this is directly related to this question I posted. 
#5066

File: dev/CommonStyles/DatePicker_themeresources.xaml [hunk 1]
```diff
@@ -177,7 +172,8 @@
                     <Grid x:Name="LayoutRoot" Margin="{TemplateBinding Padding}">
                         <Grid.Resources>
                             <Style x:Key="DatePickerFlyoutButtonStyle" TargetType="Button">
-                                <Setter Property="UseSystemFocusVisuals" Value="False" />
+                                <Setter Property="UseSystemFocusVisuals" Value="True"/>
```
Comment (marksfoster) [inline dev/CommonStyles/DatePicker_themeresources.xaml:175]: I wonder if we should use the contract test:

<contract6Present:Setter Property="UseSystemFocusVisuals" Value="True" />

Comment (kimsealls) [reply dev/CommonStyles/DatePicker_themeresources.xaml]: Updated

**Review by marksfoster — APPROVED** [review]

**Review by karenbtlai — APPROVED** [review]

**Pull request merged**